cssFiles/cardfiles/cardswithbadges.css

```diff
@@ -0,0 +1,111 @@
+.container1{
+    position: relative;
+    border: 1rem;
+    margin: 1rem;
+    width: 250px;
+    border-radius: 10px;
+    
+  }
+  
+  .ecom-img1{
+    position: relative;
+    height: 15rem;
+    width: 100%;
+    border-radius: 5px 5px 5px 5px;
+    
+  }
+  
+  .secondary-container1{
+    display: flex;
```

Comment:
A utility class can be made out of `display: flex` and can be reused whenever necessary rather than repeating the property several times